grpc-api/server/docker/build.gradle

```diff
@@ -20,7 +20,26 @@ dependencies {
   runtimeOnly 'ch.qos.logback:logback-classic:1.2.3'
 }
 
-def dockerLicenses = License.createFrom(project).syncDockerLicense().get()
+
+String GRPC_HEALTH_PROBE_VERSION = 'v0.3.1'
+TaskProvider<Task> downloadHealthProbe = tasks.register('downloadGrpcHealthProbe') {
+  Task dl ->
+    String dlUrl = "https://github.com/grpc-ecosystem/grpc-health-probe/releases/download/${GRPC_HEALTH_PROBE_VERSION}/grpc_health_probe-linux-amd64"
```

Comment:
This looks hard-coded to x86_64.  Does this cause any problems?  What if someone is developing on a Mac M1?